generative_ai/function_calling.py

```diff
@@ -59,13 +58,10 @@ def generate_function_call(project_id: str) -> GenerationResponse:
     weather_tool = Tool(
         function_declarations=[get_current_weather_func],
     )
-
+    # Start a chat session with the model
+    chat_session = model.start_chat()
```

Comment:
Lets drop this PR too.

1. `Chat session holds the chat history`, which is not the same as `Generate Content`
2. `generation_config` with `temperature` is important to use this function call API.